packages/api/src/external/carequality/patient.ts

```diff
@@ -31,26 +33,80 @@ const context = "cq.patient.discover";
 const iheGateway = makeIheGatewayAPIForPatientDiscovery();
 const resultPoller = makeOutboundResultPoller();
 
-export async function discover(patient: Patient, facilityNPI: string): Promise<void> {
+export async function discover(
+  patient: Patient,
+  facilityNPI: string,
+  forceEnabled = false
+): Promise<void> {
   const baseLogMessage = `CQ PD - patientId ${patient.id}`;
   const { log: outerLog } = out(baseLogMessage);
   const { cxId } = patient;
 
-  if (!iheGateway) return outerLog(`IHE GW not available, skipping PD`);
-  if (!(await isCQDirectEnabledForCx(cxId))) {
-    return outerLog(`CQ disabled for cx ${cxId}, skipping PD`);
+  const enabledIHEGW = await validatePDEnabled(cxId, forceEnabled, outerLog);
```

Comment:
nit: `validatePDEnabledAndInitGW` would be more accurate

I was confused as to what `enabledIHEGW` was just looking at the name